tests/Node.py

```diff
@@ -1687,3 +1722,24 @@ def waitForIrreversibleBlockProducedBy(self, producer, startBlockNum=0, retry=10
             retry = retry - 1
             startBlockNum = latestBlockNum + 1
         return False
+
+    @staticmethod
+    def parseProducers(nodeNum):
```

Comment:
it looks like you didn't change that method but moved it. can you try to get it back? just for sake of keeping history of changes.

Reply:
I moved it here since it makes more logical sense to be associated with the Node.  Right now there are no dependencies from Node on Cluster, which I think is logically what we want, since a Cluster contains a Node, so I'd like to keep it here.